EIPS/eip-4762.md

```diff
@@ -236,11 +236,13 @@ When executing a transaction, maintain four sets:
 
 When an **access** event of `(address, sub_key, leaf_key)` occurs, perform the following checks:
 
+ * If `address` is either the transaction's originator or target, skip the following steps.
  * If `(address, sub_key)` is not in `accessed_subtrees`, charge `WITNESS_BRANCH_COST` gas and add that tuple to `accessed_subtrees`.
  * If `leaf_key` is not `None` and `(address, sub_key, leaf_key)` is not in `accessed_leaves`, charge `WITNESS_CHUNK_COST` gas and add it to `accessed_leaves`
 
 When a **write** event of `(address, sub_key, leaf_key)` occurs, perform the following checks:
 
+ * If `address` is either the transaction's originator or target, skip the following steps.
```

Comment:
The way this edit was made implicates that we're not going to change CHUNK_FILL_COST in case `address` is the tx originator or target . Is that correct?
I think @gballet and @dankrad agreed to keep CHUNK_FILL_COST during the last VIC.

Reply:
yeah but not for this use case, adding elements to the witness for accessing the basic fields of `to` and `from` are not going to be charged as it's assumed to be covered by the intrinsic gas. Happy to discuss this again during the next VIC if you think I'm wrong.